app/validators/phone_number_validator.rb

```diff
@@ -0,0 +1,7 @@
+class PhoneNumberValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    unless value.remove(/\D/).match /\A1?\d{10}\z/
```

Comment:
Can this be consolidated with the regex logic in the PhoneNumberMethods Concern?

Reply:
I tried that originally. GCF follows your suggestion and does it pre-validation but that would mean that the validation error reported would have some cleaning already and not what the user provided.  
I thought it better to keep validation and normalization separate.